src/View/Calendar.php

```diff
@@ -122,6 +122,9 @@ public static function renderTable( $atts ): string {
 					true
 				);
 
+				$timeframes = array_filter($timeframes, function ($timeframe){
+					return commonsbooking_isCurrentUserAllowedToBook($timeframe->ID);
+				});
```

Comment:
@hansmorb : Kannst du mir noch einen Gefallen tun und diese Filtergeschichte in eine Helper Klasse oder einen Trait auslagern? Sollten wir irgendwann was daran anpassen müssen, brauchen wir es dann nur an einer Stelle zu tun.

Reply:
Magst du mal schauen ob das so passt? Ist jetzt leider relativ viel "boilerplate", vielleicht hätte ich das doch mal lieber in ein Parameter der get Funktionen ausgelagert, kann ich auch noch machen. Was denkst du?